pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,242 @@
+
+Using the code snippets from the previous tutorial, where you learned [how to manage Avro schemas using a centralized schema registry that enables you to share schemas across multiple applications](/docs/tutorials/kafka/managing-schemas-using-schema-registry), you will add configurations for server, operations, channel, and message bindings.
+
+Below, you can find the updated Schema reference file that we'll be using for this tutorial.
+
+```yaml
+asyncapi: 3.0.0
+info:
+  title: User Signup API
+  version: 1.0.0
+  description: The API notifies you whenever a new user signs up in the application.
+servers:
+  kafkaServer:
+    host: test.mykafkacluster.org:8092
+    description: Kafka Server
+    protocol: kafka
+operations:
+  onUserSignedUp:
+    action: receive
+    channel:
+      $ref: '#/channels/userSignedUp'
+channels:
+  userSignedUp:
+    description: This channel contains a message per each user who signs up in our application.
+    address: user_signedup
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+components:
+  messages:
+    userSignedUp:
+      payload:
+        schemaFormat: 'application/vnd.apache.avro+json;version=1.9.0'
+        schema:
+          $ref: http://localhost:8080/apis/registry/v2/groups/my-group/artifacts/UserSignedUp
+```
+
+## Add server bindings
+
+Server bindings provide protocol-specific configuration details for connecting and interacting with a server. 
+
+Server bindings allows you to specify a `schemaRegistryUrl` which provides an API Url for a given server where a schema registry was used. A Schema registry is a repository for managing and validating messages' schemas. To learn more about schema registry read [this guide on message validation](https://www.asyncapi.com/docs/guides/message-validation#schema-registry-validation).
+
+`schemaRegistryVendor` is used optionally to refer to vendors or platforms that provide the schema registry service, in this case "Apicurio".
+
+```yaml
+servers:
+  kafkaServer:
+    host: test.mykafkacluster.org:8092
+    description: Kafka Server
+    protocol: kafka
+    bindings:
+      kafka:
+        schemaRegistryUrl: 'http://localhost:8080/apis/registry/'
+        schemaRegistryVendor: 'apicurio'
+        bindingVersion: '0.4.0'
+```
+
+[Learn more about server specific bindings.](https://github.com/asyncapi/bindings/tree/master/kafka#server-binding-object).
+
+> Important: `BindingVersion` is the field version of a binding. It specifies the version of the binding specification that is used to describe how an API interacts with Kafka. The `bindingVersion` field is an optional field that is available for all bindings and ensures that as the specifications for different messaging systems evolve, APIs based on these specifications can remain consistent, compatible, and maintainable over time.  
```

Comment:
```suggestion
> Important: `bindingVersion` is the field version of a binding. It specifies the version of the binding specification that is used to describe how an API interacts with Kafka. The `bindingVersion` field is an optional field that is available for all bindings and ensures that as the specifications for different messaging systems evolve, APIs based on these specifications can remain consistent, compatible, and maintainable over time.  
```

if we refer to key name, we always have to keep original capitalization to be technically accura

Reply:
sorry but I have no idea what `The bindingVersion field is an optional field that is available for all bindings and ensures that as the specifications for different messaging systems evolve, APIs based on these specifications can remain consistent, compatible, and maintainable over time.` means. Let's not overcomplicate and add sugar in tech docs.

 ```suggestion
> Important: Always utilize the `bindingVersion` field in your production environment. This field allows you to specify the exact version of the binding you're using. Failure to include this field may result in unintended usage of the latest version, potentially causing issues due to possible future breaking changes.
```

Reply:
@derberg The intention was not to add sugar. I understood that the binding version was important for each bindings because of compatibility. And knowing the bindingVersion for each bindings ensures consistency as well. 

However, I have removed the sentence.